geometry/create-and-edit-geometries/src/main/java/com/esri/samples/create_and_edit_geometries/CreateAndEditGeometriesController.java

```diff
@@ -0,0 +1,591 @@
+   */
+  private void createGraphicsMarkers() {
+    // orange-red square for points
+    pointSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.SQUARE, Color.ORANGERED, 10);
+    // yellow circle for multipoints
+    multipointSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, Color.YELLOW, 5);
+    // thin blue line for polylines
+    lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.BLUE, 2);
+    // black outline for polygons
+    SimpleLineSymbol polygonLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DASH, Color.BLACK, 1);
+    // translucent red interior for polygons
+    fillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, Color.rgb(255, 0, 0, 0.3), polygonLineSymbol);
+  }
+
+  /**
+   * Starts the geometry editor with the point geometry type.
+   */
+  @FXML
+  private void handlePointButtonClicked() {
+    if (!geometryEditor.isStarted()) {
+      graphicsOverlay.clearSelection();
+      geometryEditor.start(GeometryType.POINT);
+
+      // disable tool selection when point or multipoint selected
+      toolComboBox.setDisable(true);
+    }
+  }
+
+  /**
+   * Starts the geometry editor with the multipoint geometry type.
+   */
+  @FXML
+  private void handleMultipointButtonClicked() {
+    if (!geometryEditor.isStarted()) {
+      graphicsOverlay.clearSelection();
+      geometryEditor.start(GeometryType.MULTIPOINT);
+
+      // disable tool selection when point or multipoint selected
+      toolComboBox.setDisable(true);
+    }
+  }
+
+  /**
+   * Starts the geometry editor with the polyline geometry type.
+   */
+  @FXML
+  private void handlePolylineButtonClicked() {
+    if (!geometryEditor.isStarted()) {
+      graphicsOverlay.clearSelection();
+      geometryEditor.start(GeometryType.POLYLINE);
+    }
+  }
+
+  /**
+   * Starts the geometry editor with the polygon geometry type.
+   */
+  @FXML
+  private void handlePolygonButtonClicked() {
+    if (!geometryEditor.isStarted()) {
+      graphicsOverlay.clearSelection();
+      geometryEditor.start(GeometryType.POLYGON);
+    }
+  }
+
+  /**
+   * Undo the last change made to the geometry while editing is active.
+   */
+  @FXML
+  private void handleUndoButtonClicked() {
+    geometryEditor.undo();
+  }
+
+  /**
+   * Redo the last change made to the geometry while editing is active.
+   */
+  @FXML
+  private void handleRedoButtonClicked() {
+    geometryEditor.redo();
+  }
+
+  /**
+   * Deletes the currently selected element of the geometry.
+   */
+  @FXML
+  private void handleDeleteSelectedElementButtonClicked() {
+    geometryEditor.deleteSelectedElement();
+  }
+
+  /**
+   * Saves the geometry in the editor to the graphics overlay and sets its symbol to the type relevant for the geometry.
+   */
+  @FXML
+  private void handleStopAndSaveButtonClicked() {
+    if (geometryEditor.getGeometry().isEmpty()) {
+      System.err.println("Geometry is empty. Unable to save geometry.");
+    }
```

Comment:
```suggestion
```
This pattern isn't something we generally do in samples so I suggest adding an Alert as per below comment